src/patternfly/demos/Tabs/tabs--table.hbs

```diff
@@ -71,7 +71,7 @@
       {{#> table-td table-td--data-label="Last commit"}}
         2 days ago
       {{/table-td}}
-      {{#> table-td table-td--action="true"}}
+      {{#> table-td table-td--IsAction=true}}
         {{> tabs--table-overflow-menu tabs--table-overflow-menu--id=(concat table--id "-dropdown-kebab-1")}}
```

Comment:
Similar to above this tabs--table-overflow-menu can be removed from the td's in this demo.